src/app/infrastructure/app-routing.module.ts

```diff
@@ -16,7 +17,8 @@ const routes: Routes = [
   {path: 'unit/:unitId/kc/:kcId', component: KnowledgeComponentComponent, canActivate: [AuthGuard], data: {role: 'learner'}},
   {path: 'analytics/events', component: EventsTableComponent, canActivate: [AuthGuard], data: {role: 'instructor'}},
   {path: 'analytics/kc-statistics', component: KcStatisticsComponent, canActivate: [AuthGuard], data: {role: 'instructor'}},
-  {path: 'analytics/:groupId/learner-progress', component: KcmProgressComponent, canActivate: [AuthGuard], data: {role: 'instructor'}},
+  {path: 'course/:courseId/learner-progress/:unitId', component: KcmProgressComponent, canActivate: [AuthGuard], data: {role: 'instructor'}},
```

Comment:
We should at least visually separate the routes between different roles. Add a blank space between the learner, instructor and general routes and in the future we can separate them into standalone lists.